modules/ROOT/pages/index.adoc

```diff
@@ -1,86 +1,130 @@
-Earlier versions of Studio include xref:3.9@mule-runtime::index.adoc[Mule 3].
+--
+[discrete]
+=== Design
 
-|===
-| **Mule 3**    | **Mule 4**
-| xref:3.9@mule-runtime::anypoint-connectors.adoc[Connectors for Mule 3] | xref:connectors::index.adoc[Connectors for Mule 4]
-| xref:3.9@mule-runtime::dataweave.adoc[DataWeave 1 for Mule 3]  | xref:mule-runtime::dataweave.adoc[DataWeave 2 for Mule 4]
-| xref:3.x@apikit::index.adoc[APIkit 3] | xref:4.x@apikit::overview-4.adoc[APIkit 4]
-| xref:3.9@connector-devkit::index.adoc[Connector DevKit] | xref:1.1@mule-sdk::index.adoc[Mule SDKs]
-|===
+Create and publish new API specifications from scratch.
+
+* xref:#[Create a reusable API specification]
+* xref:#[Plan for access]
+* xref:#[Plan for security]
+--
+
+image::develop-and-test.png[]
+
+--
+[discrete]
+=== Develop & Test
+
+Build and test APIs and integrations.
 
-== Test Integrations
+* xref:#[Develop your API]
+* xref:#[Connect your data, systems, and apps]
+* xref:#[Develop the Mule app that integrates systems, services, APIs, and devices]
+* xref:#[Build automated tests for your APIs and integrations]
+--
 
-Test the integrations you create before you deploy them.
+image::deploy.png[]
 
-* xref:2.1@munit::index.adoc[Testing with MUnit]
-* xref:runtime-manager::troubleshooting.adoc[Troubleshooting]
+--
+[discrete]
+=== Deploy
 
-== Deploy Integrations
+Deploy APIs and apps to your hosting targets, and secure them with policies.
 
-Deploy your integration into a production environment.
+* xref:#[Deployment options]
+* xref:#[Configure a proxy for your API]
+* xref:#[Create policies for your API]
+--
 
-* xref:runtime-manager::deployment-strategies.adoc[Choosing a deployment strategy]
-* xref:3.9@mule-runtime::installing.adoc#mule-installation-procedure[Installing Standalone Mule 3]
-* xref:mule-runtime::runtime-installation-task.adoc[Installing Standalone Mule 4]
+image::monitor.png[]
 
-== Manage Integrations
+--
+[discrete]
+=== Monitor
 
-Once your integration is running in production, you can monitor its performance.
+Monitor your APIs and integrations using dashboards, metrics, and visualization.
 
-* xref:monitoring::index.adoc[Anypoint Monitoring]
-* xref:api-functional-monitoring::index.adoc[API Functional Monitoring]
-* xref:runtime-manager::monitoring.adoc[Runtime Monitoring]
+* xref:#[Monitor Mule app performance]
+* xref:#[Monitor deployed API performance]
+* xref:#[Monitor deployed Mule app performance]
+--
 
-== Manage Anypoint Platform Features
+== Quick Links
 
-During development or after deployment, you can make changes to your integration.
+[#popular-topics]
+=== Popular Topics
 
-* xref:design-center::index.adoc[Design Center]
-* xref:2.x@api-manager::latest-overview-concept.adoc[API Manager]
-* xref:access-management::index.adoc[Access Management]
+* xref:#[CloudHub]
+* xref:#[Connectors]
+* xref:#[DataWeave Language]
+* xref:#[Design Center]
+* xref:#[Exchange]
+* xref:#[Mule Runtime 4.3]
```

Comment:
Not sure if this is a placeholder, but I suspect we don't want to point people to 4.3 when 4.4 is the latest even if 4.3 is currently more popular at the moment. A company goal is to get people using the latest, and we're moving away from versioning in the future. 

Same very likely goes for the Studio versions.

Reply:
This text came from the Figma design, so I think it's safe to say it was a placeholder. To solve it, I would recommend dropping the mention of the version. Once set up, the URL will resolve to the latest version.